### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by george-zakharov** [php-webdriver/php-webdriver#975] [Q&A] [answered]
Hi!
Firstly, thanks to all the maintainers of this project :)

My question is: are you going to make a new release? 1.12.1 or 1.13?
I'm very interested in commits with fixes for PHP8.1 like: https://github.com/php-webdriver/php-webdriver/blob/106303a50f817c2e87d46ea0f199bf51001b34fd/lib/Cookie.php#L214

Regards :)

**Comment by OndraM** [comment, marked as answer]
Hi, this was just released in version 1.12.1, [see changelog](https://github.com/php-webdriver/php-webdriver/blob/main/CHANGELOG.md#1121---2022-05-03).

**Reply by george-zakharov** [reply]
Great!
Thank you.
